src/pages/invitation.tsx

```diff
@@ -361,7 +360,12 @@ function Invitation() {
             {selectedRow === row.id && (
               <div 
               ref={modalRef}
-              className="absolute z-50 w-64 p-4 mt-2 overflow-hidden border border-gray-300 rounded-lg shadow-md dropdown right-4 bg-light-bg max-h-30 dark:bg-dark-bg">
+              className= 'absolute z-50 w-64 p-4 mt-2 transform -translate-y-[80%] overflow-hidden border border-gray-300 rounded-lg shadow-md dropdown right-4 bg-light-bg max-h-30 dark:bg-dark-bg'
+              // style={{
+              //  transform -translate-x-[70%] 
+              // }}
+              >
+              
```

Comment:
Romove those comment

Reply:
And remeber to add the way to test things on remote.

Reply:
You have commented out  code somewhere, can you remove them?